lib/driver.rb

```diff
@@ -89,16 +89,18 @@ def self._set_capabilities()
         :server => log_level
       },
       chrome_options: {
-        'args' => ['--start-maximized'],
-        'prefs' => {
-          'download': {
-            'prompt_for_download': false,
-            'directory_upgrade': true,
-            'default_directory': Dir.pwd
+        args: ['--start-maximized', '--privileged', '--disable-web-security'],
+        prefs: {
+          download: {
```

Comment:
Let's remove this stuff for now. We need a real solution with Selenium Grid for file downloads.  As we discussed.  

QEA-1714 will take care of that.